apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -597,12 +599,30 @@ NetSimLocalClientNode.prototype.onWireTableChange_ = function() {
  * sent to this node.
  * @private
  */
-NetSimLocalClientNode.prototype.onMessageTableChange_ = function() {
+NetSimLocalClientNode.prototype.onMessageTableChange_ = function(changeType) {
   if (!NetSimGlobals.getLevelConfig().automaticReceive) {
     // In this level, we will not automatically pick up messages directed
     // at us.  We must manually call a receive method instead.
     return;
   }
+  if (
+    changeType !== 'removeRowFromCache' &&
+    NetSimGlobals.getLevelConfig().messageGranularity ===
+      MessageGranularity.BITS
```

Comment:
Comment explaining that `MessageGranularity.BITS` implies we're using a simplex wire would be nice.